src/bin/zmtrainsa

```diff
@@ -117,6 +117,13 @@ fi
 source `dirname $0`/zmshutil || exit 1
 zmsetvars
 
+export TMPDIR=${zmtrainsa_tmp_directory:-${zimbra_tmp_directory}/zmtrainsa}
+if [ ! -d "$TMPDIR" ]; then
+    if ! mkdir "$TMPDIR"; then
```

Comment:
True, I assumed that I have to create the zmtrainsa subdirectory. I'll just make it fail if `$TMPDIR` doesn't exist.